core/src/main/java/gov/nasa/jpl/mudrod/weblog/structure/FtpLog.java

```diff
@@ -15,18 +15,25 @@
 
 import com.google.gson.Gson;
 
+import gov.nasa.jpl.mudrod.weblog.pre.ImportLogFile;
+
 import java.io.IOException;
 import java.io.Serializable;
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * This class represents an Apache access log line. See
  * http://httpd.apache.org/docs/2.2/logs.html for more details.
```

Comment:
It would be good to know which version of HTTPD is running at PO.DAAC. There is no point in us pointing to 2.2, if this is not the version used. You can see HTTPD versions at http://httpd.apache.org/docs/